home/models.py

```diff
@@ -535,11 +542,14 @@ class PageLinkPage(Page, TitleIconMixin):
         on_delete=models.SET_NULL,
     )
 
-    page = models.ForeignKey(Page, related_name='page_link_pages', on_delete=models.PROTECT)
+    page = models.ForeignKey(Page, null=True, blank=True, related_name='page_link_pages', on_delete=models.PROTECT)
+    external_link = models.URLField(null=True, blank=True)
 
     content_panels = Page.content_panels + [
+        ImageChooserPanel('image'),
```

Comment:
Just to clarify, the idea is that in general, icons can be both SVG or PNG (for footers (which are now articles in V2), PageLinkPages, and in principle also sections, etc), and they would also be displayed like icons, just without the recoloring.

Reply:
Not sure if it's easier to implement this via two fields (e.g. icon and icon_image, where one of them takes priority) or one (which could be either format), or leave it for later and stick with the warning for now.